app/controllers/coronavirus_form/business_details_controller.rb

```diff
@@ -31,7 +31,7 @@ def submit
   PAGE = "business_details"
 
   def validate_fields(business_details)
-    missing_fields = validate_mandatory_text_fields(REQUIRED_FIELDS, PAGE)
+    missing_fields = validate_mandatory_text_fields(PAGE, REQUIRED_FIELDS)
```

Comment:
alphabetising?

Reply:
Yes.  Also the other validator methods all take page first and the other arguments next.